src/open_inwoner/openzaak/models.py

```diff
@@ -238,6 +246,20 @@ class Meta:
             )
         ]
 
+    def informatieobjecttype_uuid(self):
```

Comment:
I initially also thought about a regex but you'd still need furl() to get the path, and then it basically does the same thing of 'last-non-empty-path-segment' but with regex engine involved instead of simple python.